UnitsNet.NumberExtensions.CS14/UnitsNet.NumberExtensions.CS14.csproj

```diff
@@ -0,0 +1,54 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <!-- NuGet properties -->
+  <PropertyGroup>
+    <PackageId>UnitsNet.NumberExtensions</PackageId>
+    <Version>6.0.0-pre017</Version>
+    <Authors>Andreas Gullberg Larsen, Lu Li, Jon Suda</Authors>
+    <Title>Units.NET NumberExtensions</Title>
+    <Description>Adds extension methods to number types to more easily create quantities, such as 5.Meters() instead of Length.FromMeters(5).</Description>
```

Comment:
Fix code example `5.Meters()`

Reply:
2. **Missing Documentation**
     - No README updates or documentation explaining the new package
     - Users need to know:
       - How to use the new syntax (`5.Meters` vs `Length.FromMeters(5)`)
       - C# 14 preview requirement
       - Differences from the existing NumberExtensions package

Reply:
README pushed